app/__init__.py

```diff
@@ -3,14 +3,17 @@
 from .api.V1 import version_1
 from .api.V2 import version_2
 from app import db_config
+from flask_jwt_extended import JWTManager
 
 
 def create_app():
     """Crete app fuction."""
     app = Flask(__name__)
+    app.secret_key = "brian"
```

Comment:
This secret key should be moved to dotenv file. This is a good practice.

Reply:
I created a variable secret_key inside the .env file